frontend/src/pages/About.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import {Link} from "react-router-dom";
+
+export interface IAboutPageProps {}
+const HomePage: React.FunctionComponent<IAboutPageProps> = () => {
+    return <div>
```

Comment:
Hey Dennis, since you have links in this component to navigate urself (navigate user), you could use a nav element instead of a div

Reply:
I changed this div to nav!